lib/ansible/module_utils/network/checkpoint/checkpoint.py

```diff
@@ -26,34 +26,203 @@
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
+from __future__ import (absolute_import, division, print_function)
 
-checkpoint_argument_spec = dict(auto_publish_session=dict(type='bool', default=True),
```

Comment:
plz do not, change this else if the tests written by Ricky would be using respective params `shippable-tests` will fail and we'll not be able to proceed further, please put a comment for these param as `used by checkpoint_*` modules which shall be deprecated in future.

Reply:
ok thanks